syntax="proto2";
package main_service;

option cc_generic_services = true;

message HttpRequest {};
message HttpResponse {};

service HttpService {
  rpc PageRequest(HttpRequest) returns (HttpResponse);
}
